cta_names_mappings.md

```diff
@@ -1,169 +1,621 @@
+* Department = [Unit 74455 of the Main Center of Special Technologies/GTsST](https://www.cisa.gov/uscert/ncas/alerts/aa22-110a)
+* Organisation =  [Main Directorate of the General Staff of the Armed Forces/GRU](https://www.cisa.gov/uscert/ncas/alerts/aa22-110a)
+
+---
+
+* Name = Black Energy
+
+---
+
+* Name = Grey Energy
+
+---
+
+* Name = Kamacite
+* Parent group = [Sandworm](https://arstechnica.com/information-technology/2021/02/hackers-tied-to-russias-gru-targeted-the-us-grid-for-years/)
+
+---
+
+* Name = Muddy Water
+* Full Aliases = [Cobalt Ultser, Seed Worm, TEMP.Zagros, Static Kitten](https://www.secureworks.com/blog/business-as-usual-for-iranian-operations-despite-increased-tensions), [Earth Vetala](https://www.hhs.gov/sites/default/files/iranian-threat-actors-and-healthcare.pdf)
+* Organisation = [Ministry of Intelligence of the Islamic Republic of Iran](https://www.cybercom.mil/Media/News/Article/2897570/iranian-intel-cyber-suite-of-malware-uses-open-source-tools/)
+* Country of organisation = [Islamic Republic of Iran](https://www.hhs.gov/sites/default/files/iranian-threat-actors-and-healthcare.pdf)
+
+---
+
+* Name = Gaza Cybergang
+* Full aliases = [TA402](https://thehackernews.com/2021/06/molerats-hackers-return-with-new.html), [ALUMINUM SARATOGA, Extreme Jackal, DustySky](https://www.secureworks.com/research/threat-profiles), [Moonlight](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Subgroups = [Molerats/Gaza Cybergang Group1](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37), [Desert Falcons/Gaza Cybergang Group2](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37), [Gaza Cybergang Group3](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+
+---
+
+* Name = Molerats
+* Full aliases = [Gaza Cybergang Group1](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Parent group = [Gaza Cybergang](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Campaigns = [SneakyPastes](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+
+---
+
+* Name = Desert Falcon
+* Full aliases = [Arid Viper, APT-C-23](https://malpedia.caad.fkie.fraunhofer.de/actor/aridviper), [Two-Tailed Scorpion](https://www.technadu.com/apt-c-23-re-emerged-front-scene-mygram-hacks/102145/), [Gaza Cybergang Group2](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
+* Parent group = [Gaza Cybergang](https://cybersecurity.att.com/blogs/labs-research/alien-labs-2019-analysis-of-threat-groups-molerats-and-apt-c-37)
```

Comment:
There's repetition of information by introducing both `Parent group` and `Subgroup` fields.

- You have `Desert Falcon` here with Parent group `Gaza Cybergang`
- In `Gaza Cybergang`, you have the subgroup `Desert Falcons/Gaza Cybergang Group2`
   - Falcon or Falcons?
   - Comma separated values are what I'd be using to update the accompanying JSON file. You've referred to Desert Falcon here by its name plus an alias: name/alias
   - The hyperlink is the same
- You are therefore repeating the parent-sub group relationship in two places as well as the alias name in two places. This means more places you have to manage.
- This also has been done inconsistently:
   - You have  `Parent group = APT41` in two places but no entry for APT41 itself

If it helps you see relationships quickly when looking up an entry then we can leave it, just note that for the JSON file I will not be repeating these fields (I'll probably only list subgroups and not double-back and refer to parent-groups too)

Reply:
There is repetition for parent and subgroups. This is deliberate. Lets say Candy Corn and Pink Rainbow are two groups. Candy Corn is the parent group and Pink Rainbow is the subgroup. If I look at the entry for Pink Rainbow and nothing else, I should be able to see that they are subgroup of Candy Corn. I shouldn't have to trawl through the list to come across Candy Corn and see Pink Rainbow listed as a subgroup. If nothing is listed under Pink Rainbow and that's all I look at I don't know that relationship exists.

Right now we don't need the software to know about those connections. Down the track we can leverage it but right now this is purely for human readability to understand the relationships, and to get more keywords in for searching.

While we have a whole bunch of different fields, right now all the software needs to use them for are searching keywords.

I've corrected the plural on Falcon.

I've removed the /s.

Don't stress about the hyperlinks being the same.

That is fine as far as the JSON not repeating information.

Reply:
Oh ok, thanks for the clarification. I had the perspective of data entry and management via the JSON file whereas you've explained the rationale for needing to lookup single entries at a time. I would argue that you can Ctrl+F in seconds to find relationships (and not consider this trawling through) as that's what I had to do when converting your changes to update the JSON file. But as the markdown file will be your remit, if you're happy that works for you then let's stick with it.